homeassistant/components/sensor/arest.py

```diff
@@ -157,6 +157,15 @@ def update(self):
         """Get the latest data from aREST API."""
         self.arest.update()
 
+    @property
+    def available(self):
+        """Query the device to check connection status."""
+        try:
+            requests.get(self._resource, timeout=10)
```

Comment:
No ⚠️ 

Properties on an entity should _never_ do any data fetching. All data should be cached when calling `update()`